lib/charms/mysql/v0/mysql.py

```diff
@@ -1340,15 +1371,15 @@ def set_dynamic_variable(
     def get_variable_value(self, variable: str) -> str:
         """Get the value of a variable."""
         get_var_command = [
-            f"shell.connect('{self.server_config_user}:{self.server_config_password}@{self.socket_uri}')",
+            f"shell.connect('{self.instance_def(self.server_config_user)}')",
             f"result = session.run_sql(\"SHOW VARIABLES LIKE '{variable}'\")",
             "print(result.fetch_all())",
         ]
 
         try:
             output = self._run_mysqlsh_script("\n".join(get_var_command))
         except MySQLClientError:
-            logger.exception(f"Failed to get variable {variable}")
+            logger.exception(f"Failed to get value for {variable=}")
```

Comment:
will the output be useful? The value obliviously will be empty (as failed to request).